source/Minable.cpp

```diff
@@ -45,7 +45,9 @@ void Minable::Load(const DataNode &node)
 		if(child.Token(0) == "sprite" && child.Size() >= 2)
 			SetSprite(SpriteSet::Get(child.Token(1)));
 		else if(child.Token(0) == "hull" && child.Size() >= 2)
-			hull = child.Value(1);
+			hull = max(0., child.Value(1));
+		else if(child.Token(0) == "random hull" && child.Size() >= 2)
+			randomHull = child.Value(1);
```

Comment:
Random hull should still be clamped to a positive value.
```suggestion
			randomHull = max(0., child.Value(1));
```